web/supabase/sql/AIRDROP_RESEED_RESUME.md

```
@@ -0,0 +1,178 @@
+| | |
+| --- | --- |
+| Days written | **525 / 690** (through 2026-03-19) |
+| Rows | 112,930 |
+| Remaining | 165 days, the heaviest in the dataset |
+| Blocked by | a 30s `statement_timeout` that the heavy queries now exceed |
+
+Nothing has been written to the database. No backup, staging, swap or cursor step has been run.
+
+## 0. Do not delete `web/tmp/`
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)